domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/SentenceType.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.domain.learningandworkprogress.review
+
+enum class SentenceType {
```

Comment:
I've moved this out of `ReviewService` (below) into a source file of it's own
(still not sure about the name, but moving it out of ReviewService feels like a step in the right direction)